doc(release): prepare release notes for OSS 23.10 (#3135)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.10

Release date: `March 12, 2024`

<details open>
  <summary>Enhancements</summary>

> - [Resource Status] Improved update of filters by removing useless dialog

Reviewer: ```suggestion
- [Resource Status] Improved the update of filters by removing unnecessary dialog box.
```

> - [Resource Status] Improved the update of filters by removing unnecessary dialog box.

Reviewer: ```suggestion
- [Resource Status] Improved the update of filters by removing an unnecessary dialog box.
```

</details>

<details>
  <summary>Bug fixes</summary>

- [Authentication] Fixed issue in SSO authentication.

> - [API] Fixed a visibility issue on UserFilter endpoint.

Reviewer: ```suggestion
- [API] Fixed a visibility issue with the UserFilter endpoint.
```

- [API] Fixes an issue where 500 error was returned when a host didn't have alias.

> - [CLAPI] Fixed a decoding issue on HTML entities export.

Reviewer: ```suggestion
- [CLAPI] Fixed a decoding issue affecting the export of HTML entities.
```

> - [CLAPI] While exporting services and hosts, duplicates were generated.

Reviewer: ```suggestion
- [CLAPI] Duplicates are no longer generated when exporting services and hosts.
```

> - [Dashboards - Map widget] Fixed Map, Geoview and perspective save.

Reviewer: To be improved

> - [Dashboards - Map widget] Fixed Map, Geoview and perspective save.

Reviewer: ```suggestion
- [Dashboards - Map widget] Fixed an issue where Map, Geoview and perspectives couldn't be saved.
```

> - [Dashboards - Map widget] Fixed an issue where Map, Geoview and perspectives couldn't be saved.

Reviewer: ```suggestion
- [Dashboards - Map widget] Fixed an issue where Maps, Geoviews and perspectives couldn't be saved.
```
